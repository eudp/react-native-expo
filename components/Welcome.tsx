import { StyleSheet, View } from "react-native";
import CustomText from "./CustomText";

const Welcome = () => (
  <View style={styles.container}>
    <CustomText style={styles.welcomeTitle} testID="welcome-title">
      Bienvenido de vuelta!
    </CustomText>
    <CustomText style={styles.nameSubtitle}>Ruben Rodriguez</CustomText>
  </View>
);

const styles = StyleSheet.create({
  welcomeTitle: {
    fontFamily: "Avenir-Black",
    fontSize: 20,
    lineHeight: 27.32,
    color: "#020202",
  },
  nameSubtitle: {
    fontSize: 18,
    color: "#020202",
    fontFamily: "Avenir-Regular",
  },
  container: {
    marginBottom: 20,
  },
});

export default Welcome;
