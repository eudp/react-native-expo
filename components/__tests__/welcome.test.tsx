import React from "react";
import renderer from "react-test-renderer";
import Welcome from "../Welcome";

describe("Welcome", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Welcome />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
