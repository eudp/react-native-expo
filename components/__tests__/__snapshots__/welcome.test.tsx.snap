// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome renders correctly 1`] = `
<View
  style={
    {
      "marginBottom": 20,
    }
  }
>
  <Text
    style={
      [
        {
          "fontFamily": "Avenir-Regular",
        },
        {
          "color": "#020202",
          "fontFamily": "Avenir-Black",
          "fontSize": 20,
          "lineHeight": 27.32,
        },
      ]
    }
    testID="welcome-title"
  >
    Bienvenido de vuelta!
  </Text>
  <Text
    style={
      [
        {
          "fontFamily": "Avenir-Regular",
        },
        {
          "color": "#020202",
          "fontFamily": "Avenir-Regular",
          "fontSize": 18,
        },
      ]
    }
  >
    Ruben Rodriguez
  </Text>
</View>
`;
